package.json

```diff
@@ -17,6 +17,7 @@
     "start:places-autocomplete-example": "EXAMPLE=places-autocomplete run-p start:library start:example",
     "start:directions-example": "EXAMPLE=directions run-p start:library start:example",
     "start:distance-matrix-example": "EXAMPLE=distance-matrix run-p start:library start:example",
+    "start:max-zoom-service-example": "EXAMPLE=max-zoom run-p start:library start:example",
```

Comment:
We don't have the word `service` in the other task names, so let's either add it to the other tasks as well, or remove it here.

Reply:
Was thinking of refactoring the others as well (hm). Because it would probably change when we add `autocomplete service` and then also have `autocomplete`. So, I was thinking of adding service to all the services and start here... But could also end in a quite long start script. What do you think?

Reply:
I'm ok with long task names, but let's do this identical for all tasks and remember to update the task names also in the READMEs of the examples. (This could all be done in a new pull request, if you want.)

Reply:
Okay perfect :) Yes would prefer to do the changes of the others in a new PR, as it has nothing to do with the max zoom service hook here - Updated the README here (check)